bindings/python/release.sh

```diff
@@ -66,7 +62,20 @@ elif [ "Darwin" = "$(uname -s)" ]; then
         docker run --rm -v `pwd`:/python $image /python/build-manylinux-wheel.sh
     done
 
-    # Build the mac wheel
+    # Build the x86_64 only mac wheel
+    rm -rf build libmongocrypt pymongocrypt/*.so pymongocrypt/*.dll pymongocrypt/*.dylib
+    curl -O https://s3.amazonaws.com/mciuploads/libmongocrypt-release/macos_x86_64/${BRANCH}/${REVISION}/libmongocrypt.tar.gz
+    mkdir libmongocrypt
+    tar xzf libmongocrypt.tar.gz -C ./libmongocrypt
+    NOCRYPTO_SO=libmongocrypt/nocrypto/lib/libmongocrypt.dylib
+    chmod +x ${NOCRYPTO_SO}
+    cp ${NOCRYPTO_SO} pymongocrypt/
+    rm -rf ./libmongocrypt libmongocrypt.tar.gz
+
+    $PYTHON setup.py bdist_wheel
+
+    # Build the universal mac wheel
+    PYTHON="/Library/Frameworks/Python.framework/Versions/3.10/bin/python3"
```

Comment:
@blink1073 I know this is going to conflict with your work but I want to get this in now to make the 1.3.1 release easier.

Reply:
yep, sounds good

Reply:
Here's what gets built on mac locally after these changes:
```
pymongocrypt-1.3.1-py2.py3-none-macosx_10_14_x86_64.whl
pymongocrypt-1.3.1-py2.py3-none-macosx_11_0_universal2.whl
pymongocrypt-1.3.1-py2.py3-none-manylinux2010_x86_64.whl
pymongocrypt-1.3.1-py2.py3-none-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
pymongocrypt-1.3.1.tar.gz
```